### PR TITLE
added code blocks

### PR DESCRIPTION
**Pull request by mikebz**
putting shell and code into code blocks per @frankfarzan 's request.

### CODE REVIEW

File: package-examples/helloworld-kustomize/README.md [hunk 1]
```diff
@@ -15,14 +15,17 @@ patch in it.
...
 
       fetching package /package-examples/helloworld-kustomize from https://github.com/GoogleContainerTools/kpt to helloworld-kustomize
+```
 
 ### View the package contents
 
 List the package contents in a tree structure.
 
+```shell
     $ kpt pkg tree helloworld-kustomize/
```
Comment (frankfarzan) [inline package-examples/helloworld-kustomize/README.md:29]: Remove indentation since you have block quotes.

Comment (mikebz) [reply package-examples/helloworld-kustomize/README.md]: done.

File: package-examples/helloworld/README.md [hunk 2]
```diff
@@ -13,32 +13,40 @@ doesn't do anything beyond declaring the current directory as a `kpt` package.
...
+└── [service.yaml]  Service helloworld-gke
+```
 
 ### Apply the package
 
 Initialize the inventory object:
 
-    $ kpt live init helloworld/
+```shell
+  $ kpt live init helloworld/
```
Comment (frankfarzan) [inline package-examples/helloworld/README.md:40]: remove indentation

**Comment by frankfarzan** [comment]
lgtm after fix

**Review by phanimarupaka — APPROVED** [review]
lgtm after comment from Frank

File: package-examples/helloworld/README.md (hunk 2 above)
Comment (mikebz) [reply package-examples/helloworld/README.md]: done

**Pull request merged**